pkg/controllers/cstorvolumeconfig/controller_base.go

```diff
@@ -324,3 +331,28 @@ func (c *CVCController) processNextWorkItem() bool {
 
 	return true
 }
+
+func isV1PodDisruptionBudgetSupported(client discovery.DiscoveryInterface) bool {
+	const (
+		podDisruptionBudgetKind      = "PodDisruptionBudget"
+		podDisruptionBudgetStableApi = "policy/v1"
+	)
+
+	//Checking if the policy/v1 API contains the CronJob Kind
+	apiList, err := client.ServerResourcesForGroupVersion(podDisruptionBudgetStableApi)
+	if err != nil {
+		if k8serror.IsNotFound(err) {
```

Comment:
is the indentation okay? please re-run go-fmt on save for this file

Reply:
It's 8 spaces. It's correct. Github uses 8 spaces for tabs.